docs/plugins/components.md

```diff
@@ -0,0 +1,842 @@
+---
+sidebar_position: 5
+title: Components
+description: Documented components for plugins
+---
+
+# Components
+
+:::caution
+
+Any components below whose name ends in `Item` already has a `FormItem` (or equivalent) wrapper. You
+should not use this on those components.
+
+:::
```

Comment:
Shoould be moved to Input components below the table

Reply:
But then ButtonItem which is not in the input components section, would not be part of it.

Reply:
Actually, let's put it with `FormItem`. I think that makes more sense anyways.

That reminds me, some components like `FormItem` are not exactly clear on what they do from just the name, so we should add a description to it.